content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,68 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updates to existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing resource - for example, a ConfigMap -
+using an HTTP PUT. For a PUT request, it is the client's responsibility to specify
+the `resourceVersion` (taking this from the object being updated). Kubernetes uses
+that `resourceVersion` information so that the API server can detect lost updates
+and reject requests made by a client that is out of date with the cluster.
+In the event that the resource has changed (the `resourceVersion` the clienti
+provided is stale), the API server returns a `409 Conflict` error response.
```

Comment:
I don't know if that whole paragraph properly serves its purpose. This is basically introducing [optimistic locking](https://en.wikipedia.org/wiki/Optimistic_concurrency_control), and I don't know if that description makes that clear. I would also probably mention that PUT must almost exclusively be used in a "read-modify-write" pattern which the "taking this from the object being updated" makes confusing to some extent.

Also note that, as mentioned in the Wikipedia page for optimistic locking, that mostly works for low data contention systems (and patch on the other hand helps avoid that problem, see [discussion below](https://github.com/kubernetes/website/pull/31261/files#r939597111)).

Reply:
```suggestion
provided is stale), the API server returns a `409 Conflict` error response.

A common use-case is called "read-modify-write", one would first GET the object,
unmarshal it, modify it, re-marshal it before doing a PUT to write it back. That doesn't
require to worry about that `resourceVersion` that will get carried from the GET automatically.
```

Maybe, or something like this. I think the phrasing of the first paragraph is correct, but confusing if you don't know how to use it properly, which that second paragraph is trying to address.